packages/ui/v2/core/apps/Categories.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import { ArrowRight } from "react-feather";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { AppCategories } from "@calcom/prisma/client";
+
+// TODO: See if it is worth it to move this to the database or not
+const categoriesDescription: { [key in AppCategories]: string } = {
+  calendar: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
+  messaging: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
+  other: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
+  payment: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
+  video: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
+  web3: "Ut ea id veniam commodo sit ad consequat elit ea labore dolore ex tempor nulla.",
```

Comment:
@Jaibles we need to define these texts

Reply:
Remove them for now and let's followup. Please open an issue for this

Reply:
Commented the component from Apps page, will create an issue to follow up its inclusion